documentation/user_docs/docs/manual/Binary_operations.rst

```diff
@@ -243,17 +239,17 @@ the corresponding functions are:
 Binary operations manager
 --------------------------------------------
 
-``sqw`` objects contain both pixels and image information and this information is consistent, i.e. 
+``sqw`` objects contain both pixels and image information and this information is consistent, i.e.
 image is calculated from pixels and pixels are sorted within ``PixelData`` array in such a way that the block of
 pixels contributed into image bin(cell) is located in specific position of ``PixelData`` array and this position can be
-identified from image. The position :math:`i_1` of the first pixel contributing into image bin(cell) number :math:`n` is defined by
-formula: :math:`i_1 = cumsum(sqw.data.npix(1:n-1))+1` and the last by: :math:`i_{end} = i_1+sqw.data.npix(n)-1` where 
-:math:`sqw.data.npix` refers to ``npix`` array of ``dnd`` object. Particular pixels positions between :math:`i_1` and :math:`i_{end}`
-are random. 
+identified from image. The position :code:`i_1` of the first pixel contributing into image bin(cell) number :code:`n` is defined by
+formula: :code:`i_1 = cumsum(sqw.data.npix(1:n-1))+1` and the last by: :code:`i_{end} = i_1+sqw.data.npix(n)-1` where
+:code:`sqw.data.npix` refers to ``npix`` array of ``dnd`` object. Particular pixels positions between :code:`i_1` and :code:`i_{end}`
```

Comment:
Rather than taking the code role, do these just want to be verbatim (` `` `)?

Reply:
Could do... the old code was `:math:` which did not look good, and `s/math/code/` was easier to do...

Reply:
```suggestion
identified from image. The position ``i_1`` of the first pixel contributing into image bin(cell) number ``n`` is defined by
formula: ``i_1 = cumsum(sqw.data.npix(1:n-1))+1`` and the last by: ``i_{end} = i_1+sqw.data.npix(n)-1`` where
``sqw.data.npix`` refers to ``npix`` array of ``dnd`` object. Particular pixels positions between ``i_1`` and ``i_{end}``
```